### PR TITLE
Add Link component

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Create a Link component, anchor tag.

#### What problem is this solving?
No Link component.

#### How should this be manually tested?
Pull & `yarn styleguide`

#### Screenshots or example usage
Screen Shot 2019-07-11 at 12 16 08

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by cmdalbem** [comment]
@thaynamatos vc tá vendo o design desse né?

**Comment by thaynamatos** [comment]
tô sim @cmdalbem

**Comment by thaynamatos** [comment]
Hey, @kevinch! Design done and approved in our meeting this morning. I'm going to update our Figma Library soon. But here it is: https://www.figma.com/file/aVkZXtwn3LzDdBmlUmqNnM1b/Link-component?node-id=0%3A1

File: react/components/Link/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import ExternalLink from '../icon/ExternalLink'
+
+class Link extends PureComponent {
```
Comment (salesfelipe) [inline react/components/Link/index.js]: Why not a function component?

Comment (kevinch) [reply react/components/Link/index.js]: Updated.

**Pull request merged**